repo_guidelines.md

```diff
@@ -1,5 +1,7 @@
 # Project Repo Guidelines & Template
 
+A template repo is available at https://github.com/edgi-govdata-archiving/Template. When creating a new repo, you can use it as a template to automatically apply the below settings (you will need to customize template text to fit your purpose).
```

Comment:
In some views, the period is getting included in the link (seems like it does in the source view, and does not in the rendered view). Given this is one of the oh-so-fun-and-very-non-standardized-between-markdowns things, maybe better to use link syntax with square brackets to make it explicit? Alternatively, you could probably put a zero-width space (`\u200b`) between the URL and the period.

Reply:
oh whoops, it's weird that I didn't do that. Thanks for the catch.

Reply:
Ha, I only caught it because I clicked the link in the diff view and got a 404 page. Otherwise I would have completely missed it, too. :)